src/openms/source/CHEMISTRY/AASequence.cpp

```diff
@@ -80,6 +82,70 @@ namespace OpenMS
     return *this;
   }
 
+    // Formulae that need to be added to the internal residues to get to fragment type
+    // Formulae from http://www.matrixscience.com/help/fragmentation_help.html
+    static const EmpiricalFormula& getInternalToFull()
+    {
+      static const EmpiricalFormula to_full = EmpiricalFormula("H2O");
+      return to_full;
```

Comment:
I am a bit confused, dont we already have the same code in Resudue.h so why duplicate it here? Am I missing something? 
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/include/OpenMS/CHEMISTRY/Residue.h#L70